tests/rptest/tests/timequery_test.py

```diff
@@ -359,6 +359,11 @@ def set_up_cluster(self, cloud_storage: bool, batch_cache: bool, spillover: bool
         else:
             self.redpanda.add_extra_rp_conf({"log_segment_size": self.log_segment_size})
 
+        # `test_timequery_with_local_gc` attempts to set the `log_segment_size` lower than the minimum
+        # value of 1_MiB- disable bounded property checks to allow this.
+        self.redpanda.set_environment(
+            {"__REDPANDA_TEST_DISABLE_BOUNDED_PROPERTY_CHECKS": "ON"}
+        )
```

Comment:
ahh the warning was on bootup, not from something later like alter topic config. i think the later will fail, but indeed this just seems to warn silently on bootup.